packages/l10n/src/create-l10n-injector/create-l10n-injector.spec.tsx

```diff
@@ -153,7 +153,7 @@ describe('when there is an error loading L10n data', () => {
       propLoadingKey: 'l10nInjector',
       loadLocale: loadLocalesErrorMock,
     });
-    const WrappedComponent: React.ComponentClass<any> = l10nInjector(
+    const WrappedComponent: React.ComponentType<any> = l10nInjector(
```

Comment:
I am still never sure when to say `any` or if that's a good idea. I read quite a few articles back then and felt having `Props extends object` can't hurt. Also if you check the typing of React and what it defaults to it's a `{}`. Props extends object

Reply:
Yeah, but if possible we should always opt to having proper types so that we can make sure the compile has our back.